lib/gs1/validations/length_validation.rb

```diff
@@ -0,0 +1,25 @@
+module GS1
+  module Validations
+    # Ensures correct length validation.
+    #
+    module LengthValidation
+      def validate_length(options = {})
+        errors << 'Invalid length' unless valid_length?(options)
+      end
+
+      def valid_length?(options)
+        normalize_allowed(options[:allowed]).include?(data&.size)
+      end
+
+      private
+
+      def normalize_allowed(allowed)
+        case allowed
+        when Range then allowed.to_a
+        when Array then allowed.flatten
```

Comment:
When would you need this?

Reply:
In case someone types `[[5]]`

Reply:
I can change, seems dumb when you say it :D